src/components/Tools/PaperSize.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Label, Icon, Button } from 'semantic-ui-react';
+import uuid from 'uuid/v4';
+import { changePaperSize } from '../../actions/app.actions';
+// import fonts from '../../helpers/font.helper';
+
+export const paperSizes = [
+  { tag: 'letter', name: 'Letter (8.5" x 11")' },
+  { tag: 'a4', name: 'A4 (210mm × 297mm)' },
+  { tag: 'legal', name: 'Legal (8.5" x 14")' },
+];
+
+class PaperSize extends Component {
+  constructor(props) {
+    super(props);
+    this.handlePaperSizeChange = this.handlePaperSizeChange.bind(this);
+  }
+
+  handlePaperSizeChange(e) {
+    const { dispatch } = this.props;
+    const selection = e.target.selectedIndex;
+    const paperSize = paperSizes[selection].tag;
+    dispatch(changePaperSize(paperSize));
+  }
+
+  render() {
+    const { paperSize } = this.props;
+
+    return (
+      <div className="json-resume-tool font-selector">
+        <Label size="big" basic>
+          <Icon name="font" />
+          Paper Size
+        </Label>
+        <Button
+          size="large"
+          fluid
+          style={{
+            display: 'block',
+            backgroundColor: 'white',
+            color: 'black',
+          }}
+        >
+          <select
+            id="font-selector"
```

Comment:
ID should be unique.

Reply:
DOH!

Reply:
As far I could tell, `#font-selector` was only being used as a CSS selector, so I replaced it with `.options-selector` in both PaperSize.js and FontSelector.js